### PR TITLE
fix(unsupported): Detail fixes

### PR DESCRIPTION
**Pull request by janpio**
Some detail fixes for the Unsupported mentions.

### CODE REVIEW

File: content/400-reference/200-api-reference/100-prisma-schema-reference.mdx [hunk 1]
```diff
@@ -994,17 +994,17 @@ Not supported
...
 
-The `Unsupported` type was introduced in [2.17.0](https://github.com/prisma/prisma/releases/tag/2.17.0) and allows you to represent data types in the Prisma schema that are not supported by the Prisma Client. `Unsupported` fields can be introspected with `prisma db pull` or created with Prisma Migrate or `db push`.
+The `Unsupported` type was introduced in [2.17.0](https://github.com/prisma/prisma/releases/tag/2.17.0) and allows you to represent data types in the Prisma schema that are not supported by Prisma Client. Fields of type `Unsupported` can be created during Introspection with `prisma db pull` or written by hand, and created in the database with Prisma Migrate or `db push`.
 
 #### Remarks
 
 - Fields with `Unsupported` types are not available in the generated client.
 - If a model contains a **mandatory** `Unsupported` type, `prisma.model.create(..)` and `prisma.model.update(...)` are not available in the client.
-- Prisma will always warn that your schema contains unsupported types when you `introspect`:
+- When introspecting a databases that contains unsupported types, Prisma will always warn:
```
Comment (keerlu) [inline content/400-reference/200-api-reference/100-prisma-schema-reference.mdx]:
```suggestion
- When you introspect a database that contains unsupported types, Prisma will provide the following warning:
```

File: content/400-reference/200-api-reference/100-prisma-schema-reference.mdx [hunk 2]
```diff
@@ -994,17 +994,17 @@ Not supported
 <Admonition type="warning">
 
 **Not supported by MongoDB** <br />
-The [MongoDB connector](/concepts/database-connectors/mongodb) does not currently support the `Unsupported` type.
+The [MongoDB connector](/concepts/database-connectors/mongodb) does not support the `Unsupported` type.
 
 </Admonition>
 
-The `Unsupported` type was introduced in [2.17.0](https://github.com/prisma/prisma/releases/tag/2.17.0) and allows you to represent data types in the Prisma schema that are not supported by the Prisma Client. `Unsupported` fields can be introspected with `prisma db pull` or created with Prisma Migrate or `db push`.
+The `Unsupported` type was introduced in [2.17.0](https://github.com/prisma/prisma/releases/tag/2.17.0) and allows you to represent data types in the Prisma schema that are not supported by Prisma Client. Fields of type `Unsupported` can be created during Introspection with `prisma db pull` or written by hand, and created in the database with Prisma Migrate or `db push`.
```
Comment (keerlu) [inline content/400-reference/200-api-reference/100-prisma-schema-reference.mdx:1017]: I may not be understanding how this works at all, so please ignore any part of this suggestion that doesn't make sense...

```suggestion
The `Unsupported` type was introduced in [2.17.0](https://github.com/prisma/prisma/releases/tag/2.17.0) and allows you to represent data types in the Prisma schema that are not supported by Prisma Client. You can create fields of type `Unsupported` when you introspect with `db pull`, or add them manually to your Prisma schema and apply them to the database with Prisma Migrate or `db push`.
```

**Review by keerlu — APPROVED** [review]
Looks good, couple of small copyedits

File: content/400-reference/200-api-reference/100-prisma-schema-reference.mdx (hunk 2 above)
Comment (janpio) [reply content/400-reference/200-api-reference/100-prisma-schema-reference.mdx:1017]: Yeah I struggled with this sentence.

This is fundamentally about 2 directions:
1. Getting them into the schema, which can be done via Introspection or manually
2. Migrating them to a database when you just have the schema, which is Migrate or `db push`.

Maybe 2 separate sentences would be better here.

**Comment by janpio** [comment]
I think the individual changes here are still useful and should be applied to the current state of the documentation of this.

**Pull request merged**